JeMPI_Apps/JeMPI_BackupRestoreAPI/src/main/java/org/jembi/jempi/backuprestoreapi/AppConfig.java

```diff
@@ -0,0 +1,124 @@
+package org.jembi.jempi.backuprestoreapi;
+
+import com.typesafe.config.Config;
+import com.typesafe.config.ConfigFactory;
+import org.apache.logging.log4j.Level;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import java.io.File;
+import java.util.Arrays;
+
+public final class AppConfig {
+
+   private static final Logger LOGGER = LogManager.getLogger(AppConfig.class);
+   private static final Config SYSTEM_PROPERTIES = ConfigFactory.systemProperties();
+   private static final Config SYSTEM_ENVIRONMENT = ConfigFactory.systemEnvironment();
+   public static final Config CONFIG = new Builder().withSystemEnvironment()
+                                                    .withSystemProperties()
+                                                    .withOptionalRelativeFile("/conf/server.production.conf")
+                                                    .withOptionalRelativeFile("/conf/server.staging.conf")
+                                                    .withOptionalRelativeFile("/conf/server.test.conf")
+                                                    .withResource("application.local.conf")
+                                                    .withResource("application.conf")
+                                                    .build();
+   public static final String POSTGRESQL_IP = CONFIG.getString("POSTGRESQL_IP");
+   public static final Integer POSTGRESQL_PORT = CONFIG.getInt("POSTGRESQL_PORT");
+
+   public static final String POSTGRESQL_USER = CONFIG.getString("POSTGRESQL_USER");
+   public static final String POSTGRESQL_PASSWORD = CONFIG.getString("POSTGRESQL_PASSWORD");
+   public static final String POSTGRESQL_NOTIFICATIONS_DB = CONFIG.getString("POSTGRESQL_NOTIFICATIONS_DB");
+   public static final String POSTGRESQL_AUDIT_DB = CONFIG.getString("POSTGRESQL_AUDIT_DB");
+   public static final String KAFKA_BOOTSTRAP_SERVERS = CONFIG.getString("KAFKA_BOOTSTRAP_SERVERS");
+   public static final String KAFKA_APPLICATION_ID = CONFIG.getString("KAFKA_APPLICATION_ID");
```

Comment:
@sushantpatil1214 do we need this config

Reply:
As we earlier discussed, In future we have to use PostgreSQL DB, so added this